packages/suite/src/components/suite/Timer.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import formatDuration from 'date-fns/formatDuration';
+import { useLocales } from '@suite-hooks/useLocales';
+
+const formatTimeLeft = (deadline: Date, locale?: Locale) => {
```

Comment:
Please write a unit test for this.

Reply:
553294c65ebc0f98cdd168c80c14542ad398e998